wizard/wizard_poweremail_preview.py

```diff
@@ -99,7 +100,18 @@ def action_generate_static_mail(self, cr, uid, ids, context=None):
                     field_value = template.file_name
                 else:
                     field_value = getattr(template, "def_{}".format(field))
-                vals[field] = get_value(cr, uid, record_id, field_value, template, ctx)
+
+                if field == 'body_text':
+                    body_text = get_value(
+                        cr, uid, record_id, field_value, template, ctx
+                    )
+
+                    if template.inline:
+                        vals[field] = transform(body_text)
+                    else:
+                        vals[field] = body_text
+                else:
+                    vals[field] = get_value(cr, uid, record_id, field_value, template, ctx)
```

Comment:
simpfifica més aquest codi:
```suggestion
                vals[field] = get_value(cr, uid, record_id, field_value, template, ctx)
                if field == 'body_text' and template.inline:
                        vals[field] = transform(vals[field])
```